modules/admin/proc-rhdh-deployment-config.adoc

```diff
@@ -96,6 +98,59 @@ spec:
                   claimName: dynamic-plugins-root
 ----
 
+Starting from v1alpha3 (Operator version 0.4.0), you can mount directories from pre-created PVCs using the `spec.application.extraFiles.pvcs` field. PVCs are mounted as directories to the container's path, based on the following logic:
+
+* If `spec.application.extraFiles.pvcs[].mountPath` is defined, the PVC is mounted to the specified path.
+* If `spec.application.extraFiles.pvcs[].mountPath` is not defined, the PVC is mounted under `spec.application.extraFiles.mountPath/`.
+* If neither is defined, the PVC defaults to `/opt/app-root/src`.
+
+.Example PVC objects in the namespace
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myclaim1
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myclaim2
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi
+----
+
+The PVCs in the previous example can be mounted to the container as follows:
+
+.Example PVCs mounted to the container
+[source,yaml]
+----
+spec:
```

Comment:
Would it make sense to also show the API version, Kind and metadata in this example?
```suggestion
apiVersion: rhdh.redhat.com/v1alpha3
kind: Backstage
metadata:
  name: developer-hub
spec:
```

Reply:
Well, it is two different "styles" IMO 
for upstream I'd prefer fragment, this way it is clear where to put it to existed manifest w/o redundant information.

Reply:
Yeah, I agree with the different styles, but wanted this to be consistent with the rest of the examples listed in this section: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-702/admin-rhdh/#proc-rhdh-deployment-config_assembly-admin-templates
(And having at least the API Version lets users know what is expected, I think)